base.d.ts

```diff
@@ -136,9 +137,15 @@ export type ParseOptions = {
 
 	queryString.parse('foo=1', {parseNumbers: true});
 	//=> {foo: 1}
+
+	queryString.parse('foo=1&bar=2', {parseNumbers: {includes: ['foo']}});
+	//=> {foo: 1, bar: '2'}
+
+	queryString.parse('foo=1&bar=2', {parseNumbers: {excludes: ['foo']}});
+	//=> {foo: '1', bar: 2}
 	```
 	*/
-	readonly parseNumbers?: boolean;
+	readonly parseNumbers?: boolean | {includes?: string[]; excludes?: string[]};
```

Comment:
This feels too limiting. I would prefer a filter method that receives the key and value and returns true whether it should be parsed as a number. But the ideal solution would be something like https://github.com/sindresorhus/query-string/issues/210

Reply:
Got it. I'll look at #249 remaining unfinished parts

Reply:
If you decide to work no #249, I would recommend posting a proposal on what to do in #249 before writing any code to make sure it's the most optimal solution.